pkg/aws/core_test.go

```diff
@@ -50,6 +50,9 @@ var _ = Describe("MachineServer", func() {
 			"userData":                []byte("dummy-user-data"),
 		},
 	}
+	annotations := map[string]string{
+		awsPlacement: "{ \"affinity\": \"host\", \"availabilityZone\": \"eu-west-1a\", \"tenancy\": \"host\"}",
```

Comment:
```suggestion
		awsPlacement: `{ "affinity": "host", "availabilityZone": "eu-west-1a", "tenancy": "host"}`,
```

Reply:
yes ,this way looks much cleaner , thnx :)